### ISSUE TITLE
Videos of movements on artists component

### ISSUE THREAD

**Issue by BernhardHumm** [hochschule-darmstadt/openartbrowser#295]
**Reason**
Videos about movements are displayed on artwork components (if the artwork has this movement) but not on the artist component (if the artist belongs to this movement).
Example: https://openartbrowser.org/en/artist/Q5592

**Solution**
Add video about movement on artist component if video is available and artist belongs to this movement.

**Alternatives**

**Effects**
More relevant multimedia content

**Acceptance criteria**

**Additional context**

**Comment by EeveesEyes** [comment]
Not sure if this is rather a bug than a feature because eg. [Alfred Sisley](https://openartbrowser.org/en/artist/Q175130) has a impressionism-video assigned.

**Comment by stkeogie** [comment]
We have examined the whole thing and found that it is not a bug. If an artist belongs to a movement for which a video is available, the video will also be displayed for the artist. If an artist belongs to more than one movement, only the video of the first movement will be shown.

Here is an example:
[Jean-François Raffaëlli](https://openartbrowser.org/en/artist/Q658862) belongs to several movements. The video that is displayed belongs to the Impressionism movement because it is the first in the list.


[Stefano di Giovanni](https://openartbrowser.org/en/artist/Q362016) belongs to the Sienese School & Renaissance movement. Here you should see the video of the Sienese School movement, but since there is no video of the movement, no video is shown.

To summarize, if there is a video of the movement and the artist belongs to the movement, it will be shown if it is in the first place of the artist's movements.

**Comment by stkeogie** [comment]
After a discussion in the team we decided to show multiple videos of the movements the artist belongs to, if they are available. We limit the number of videos that are shown to the user to 3, as more videos could be confusing. If a video is assigned to an artist, the video of the artist and a up to 2 movement videos will be displayed. No videos are displayed multiple times on the same Page.
